src/Altinn.Broker/Models/Webhooks/MalwareScan/EventMessage.cs

```diff
@@ -0,0 +1,14 @@
+namespace Altinn.Broker.Webhooks.Models
+{
+    public class EventMessage
+    {
+        public string id { get; set; } = string.Empty;
```

Comment:
Should capitalize the field names. The default JSON serialization config will make it work even if they are lowercase in the json.

Reply:
Field names have now been capitalized.